src/functions/delimsizing.js

```diff
@@ -210,18 +209,6 @@ defineFunction({
             }
         }
 
-        const lastChildType = html.getTypeOfDomTree(inner[inner.length - 1]);
-        const activeSpacings = options.style.isTight() ? tightSpacings : spacings;
-
-        if (lastChildType && activeSpacings[lastChildType]["mclose"]) {
-            const glue =
-                buildCommon.makeSpan(["mord", "rule"], [], options);
-            const dimension =
-                calculateSize(activeSpacings[lastChildType]["mclose"], options);
-            glue.style.marginRight = `${dimension}em`;
-            inner.push(glue);
```

Comment:
Now this isn't included in `inner`/`children`.  Won't that affect the results of `getOutermostNode`?

Reply:
@kevinbarabash The `html.buildExpression` inserts the appropriate spacing between the last child and `mclose`.

Reply:
That's pretty cool.  Glad you were able to figure out a way to get all the spacing code in one place.